### PR TITLE
Update ADOPTERS.md

### PR DESCRIPTION
**Pull request by michmike**
updating some of the wording

### CODE REVIEW

**Comment by jonasrosland** [comment]
LGTM!

**Comment by ashish-amarnath** [comment]
@michmike looks like you need to sign your commits. We can merge this once this check passes

File: ADOPTERS.md [hunk 1]
```diff
@@ -41,17 +41,18 @@ Red Hat has developed the [Cluster Application Migration Tool][51] which uses [V
...
-directory in this repo and submit a pull request with your change.
-Name the image file something that reflects your company
-(e.g., if your company is called Acme, name the image acme.png).
-We will follow up and make the change in the [velero.io][2] website.
+If you are using Velero and would like to be included in the `Velero Adopters`, add an SVG version of your logo to the site/img/adopters directory in this repo and submit a [pull request][3] with your change. Name the image file something that reflects your company (e.g., if your company is called Acme, name the image acme.png). See this for an example [PR][4].
 
-[1]: https://github.com/vmware-tanzu/velero/pulls
+### Adding a logo to velero.io
+
+If you would like to add your logo to a future `Adopters of Velero` section of [velero.io][2], follow the steps above to add your organization to the list of Velero Adopters. Our community will follow up and make the change in the [velero.io][2] website.
```
Comment (ashish-amarnath) [inline ADOPTERS.md]:
```suggestion
If you would like to add your logo to a future `Adopters of Velero` section on [velero.io][2], follow the steps above to add your organization to the list of Velero Adopters. Our community will follow up and make the change in the [velero.io][2] website.
```

File: ADOPTERS.md [hunk 2]
```diff
@@ -41,17 +41,18 @@ Red Hat has developed the [Cluster Application Migration Tool][51] which uses [V
...
-## Adding a logo to velero.io
+## Adding your organization to the list of Velero Adopters
 
-If you would like to add your logo to a future `Adopters of Velero` section
-of [velero.io][2], add an SVG version of your logo to the site/img/adopters
-directory in this repo and submit a pull request with your change.
-Name the image file something that reflects your company
-(e.g., if your company is called Acme, name the image acme.png).
-We will follow up and make the change in the [velero.io][2] website.
+If you are using Velero and would like to be included in the `Velero Adopters`, add an SVG version of your logo to the site/img/adopters directory in this repo and submit a [pull request][3] with your change. Name the image file something that reflects your company (e.g., if your company is called Acme, name the image acme.png). See this for an example [PR][4].
```
Comment (ashish-amarnath) [inline ADOPTERS.md]:
```suggestion
If you are using Velero and would like to be included in the `Velero Adopters`, add an SVG version of your logo to the `site/img/adopters` directory in this repo and submit a [pull request][3] with your change. Name the image file something that reflects your company (e.g., if your company is called Acme, name the image acme.png). See this for an example [PR][4].
```

**Comment by michmike** [comment]
i am using the GitHub UI and can't sign my commits. one of the admins of the project can simply set DCO to pass. i accepted the updates by Ashish

File: ADOPTERS.md [hunk 3]
```diff
@@ -41,17 +41,18 @@ Red Hat has developed the [Cluster Application Migration Tool][51] which uses [V
...
-directory in this repo and submit a pull request with your change.
-Name the image file something that reflects your company
-(e.g., if your company is called Acme, name the image acme.png).
-We will follow up and make the change in the [velero.io][2] website.
+If you are using Velero and would like to be included in the `Velero Adopters`, add an SVG version of your logo to the `site/img/adopters` directory in this repo and submit a [pull request][3] with your change. Name the image file something that reflects your company (e.g., if your company is called Acme, name the image acme.png). See this for an example [PR][4].
 
-[1]: https://github.com/vmware-tanzu/velero/pulls
+### Adding a logo to velero.io
+
+If you would like to add your logo to a future `Adopters of Velero` section on [velero.io][2], follow the steps above to add your organization to the list of Velero Adopters. Our community will follow up and publish it to [velero.io][2] website.
```
Comment (carlisia) [inline ADOPTERS.md]:
```suggestion
If you would like to add your logo to a future `Adopters of Velero` section on [velero.io][2], follow the steps above to add your organization to the list of Velero Adopters. Our community will follow up and publish it to the [velero.io][2] website.
```

File: ADOPTERS.md [hunk 4]
```diff
@@ -41,17 +41,18 @@ Red Hat has developed the [Cluster Application Migration Tool][51] which uses [V
...
-## Adding a logo to velero.io
+## Adding your organization to the list of Velero Adopters
 
-If you would like to add your logo to a future `Adopters of Velero` section
-of [velero.io][2], add an SVG version of your logo to the site/img/adopters
-directory in this repo and submit a pull request with your change.
-Name the image file something that reflects your company
-(e.g., if your company is called Acme, name the image acme.png).
-We will follow up and make the change in the [velero.io][2] website.
+If you are using Velero and would like to be included in the `Velero Adopters`, add an SVG version of your logo to the `site/img/adopters` directory in this repo and submit a [pull request][3] with your change. Name the image file something that reflects your company (e.g., if your company is called Acme, name the image acme.png). See this for an example [PR][4].
```
Comment (carlisia) [inline ADOPTERS.md]:
```suggestion
If you are using Velero and would like to be included in the list of `Velero Adopters`, add an SVG version of your logo to the `site/img/adopters` directory in this repo and submit a [pull request][3] with your change. Name the image file something that reflects your company (e.g., if your company is called Acme, name the image acme.png). See this for an example [PR][4].
```

File: ADOPTERS.md [hunk 5]
```diff
@@ -41,17 +41,18 @@ Red Hat has developed the [Cluster Application Migration Tool][51] which uses [V
...
 
-[1]: https://github.com/vmware-tanzu/velero/pulls
+### Adding a logo to velero.io
+
+If you would like to add your logo to a future `Adopters of Velero` section on [velero.io][2], follow the steps above to add your organization to the list of Velero Adopters. Our community will follow up and publish it to [velero.io][2] website.
+
+[1]: #adding-a-logo-to-veleroio
```
Comment (carlisia) [inline ADOPTERS.md]: I assume this ignores the `.`. If not we can fix it later.

File: ADOPTERS.md (hunk 3 above)
Comment (jonasrosland) [reply ADOPTERS.md]: LGTM

File: ADOPTERS.md (hunk 4 above)
Comment (jonasrosland) [reply ADOPTERS.md]: LGTM

**Comment by michmike** [comment]
> Only the new minor changes from Carlisia should be needed, otherwise everything looks good.

made the changes. thanks

**Comment by skriss** [comment]
Looks like we've got a small merge conflict here due to some additional adopters being added since this PR was opened

**Comment by skriss** [comment]
@carlisia can you approve as well if LGTY?

**Pull request merged**